actions/transactions/internal/mapping/outline/outline_to_response.go

```diff
@@ -0,0 +1,17 @@
+package outline
+
+import (
+	"github.com/bitcoin-sv/spv-wallet/engine/transaction/draft"
+	model "github.com/bitcoin-sv/spv-wallet/models/transaction"
+	"github.com/mitchellh/mapstructure"
+)
+
+// ToResponse converts a draft transaction to a response model.
+func ToResponse(tx *draft.Transaction) *model.AnnotatedTransaction {
+	res := &model.AnnotatedTransaction{}
+	err := mapstructure.Decode(tx, res)
+	if err != nil {
+		panic(err)
```

Comment:
Why panic here and not return err from the function?

Reply:
My thought was that, it shouldn't have happend ever.
but you're right we should return clear error, to make it easier to find the reason and fix.

Reply:
done